integration-tests/tests/src/schemas/person-and-dog-with-object-ids.ts

```diff
@@ -19,10 +19,9 @@
 /* tslint:disable max-classes-per-file */
 
 import Realm from "realm";
-import { ObjectId } from "bson";
 
 export interface IPerson {
-    _id: ObjectId;
+    _id: Realm.BSON.ObjectId;
```

Comment:
Would it be possible to make `ObjectId` a named import from `realm`?
Then we could import it so:
`import Realm, { ObjectId } from "realm";`

Reply:
I hope that's a goal for the future. Not really possible atm.

Reply:
Me too.  That would definitely be more ergonomic.